homework/src/components/SelectedPostContainer/SelectedPostContainer.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from "react";
+import { IPosts } from "src/types/types";
+import SelectedPost from "./SelectedPost/SelectedPost";
+import './style.css'
+
+const SelectedPostContainer = () => {
+  const [posts, setPosts] = useState([]);
+  useEffect(() => {
+    const getPosts = async () => {
```

Comment:
See the comment above. By the way, you can pass a prop limit to the function and fetch a specific number of posts based on your need.

const fetchPosts(limit?: number) => {
  await fetch(`https://studapi.teachmeskills.by/blog/posts/?limit=${limit}`)
  // the rest of the code
}